### DISCUSSION TITLE
what does kyuubi session Duration mean in yarn website?

### DISCUSSION THREAD

**Discussion by madeirak** [apache/kyuubi#6276] [Q&A] [answered]
as image shows 
<img width="961" alt="image" src="https://github.com/apache/kyuubi/assets/23215922/8865d235-652a-43de-be54-e4ef8b2c53ed">
![image](https://github.com/apache/kyuubi/assets/23215922/ddc49b28-674d-4c43-b302-4fb8c175cca4)
<br />
I didn't explicitly call `kyuubi_session.close()`, but I found "Finish TIme" and "Duration" on the page, 
I noticed that the official document of kyuubi mentions that session is disposable.
**I'm not sure if kyuubi session terminated on its own?**
<br />
![image](https://github.com/apache/kyuubi/assets/23215922/f93a83fe-a102-48b9-b5bd-30edd5503eff)

**Comment by pan3793** [comment]
> I didn't explicitly call `kyuubi_session.close()`, but I found "Finish TIme" and "Duration" on the page

By default, Kyuubi will close the idle session to avoid resource leaks.

> `kyuubi.session.idle.timeout` PT6H (default value), docs: session idle timeout, it will be closed when it's not accessed for this duration

**Reply by madeirak** [reply]
thx for reply, another thing I want to confirm is the "Finish TIme" and "Duration" on yarn page, if it has a value，it means the related kyuubi session is really closed/finished ?
<img width="483" alt="image" src="https://github.com/apache/kyuubi/assets/23215922/387d9e37-ea2a-4ada-a31b-4ced1dac7ee6">

Is "finish" same as "close" on kyuubi session?

**Reply by madeirak** [reply, marked as answer]
https://github.com/apache/kyuubi/issues/5894
